terraform/azure/modules/instances/variables.tf

```diff
@@ -0,0 +1,82 @@
+variable "user" {
+  description = "Instance user name that will used for SSH operations"
```

Comment:
we can remove all default values from here, as we already have 2 layers for default values.
- From plan
- From module level variable